scaffoldmaker/utils/tubemesh2.py

```diff
@@ -0,0 +1,356 @@
+'''
+Utility function for generating tubular mesh from a central line
+using a unit profile.
+Created on 18 Feb, 2019
+
+@author: Mabelle Lin
+'''
+from __future__ import division
+import math
+from scaffoldmaker.utils.eftfactory_bicubichermitelinear import eftfactory_bicubichermitelinear
+from scaffoldmaker.utils.eftfactory_tricubichermite import eftfactory_tricubichermite
+from scaffoldmaker.utils.zinc_utils import *
+from scaffoldmaker.utils.geometry import *
+from scaffoldmaker.utils.interpolation import *
+from scaffoldmaker.utils.vector import *
+from opencmiss.zinc.element import Element, Elementbasis
+from opencmiss.zinc.field import Field
+from opencmiss.zinc.node import Node
+
+def generatetubemesh(region,
```

Comment:
Note I haven't studied the implementation of this function, just the interface (which is fine for now).
I'm happy to bring it in as-is now, but will look at it in more detail when additional complications for variable wall thickness are added.